components/ws-proxy/BUILD.yaml

```diff
@@ -52,6 +53,7 @@ packages:
       - components/registry-facade-api/go:lib
       - components/supervisor-api/go:lib
       - components/ws-manager-api/go:lib
+      - components/server/go:lib
```

Comment:
The deploy time of `ws-proxy` and `server` is different on PAYG.

But considering that this is in addition to https://github.com/gitpod-io/gitpod/pull/19773 , we can ignore this problem